src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -3,9 +3,11 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Diagnostics;
 using System.Net;
 using System.Net.Http;
 using System.Net.Tests;
+using System.Reflection;
```

Comment:
As mentioned in another comment-- our preference is to continue with the expectation that project N will emit metadata for anonymous types and then file a bug against them, for instance, if the HttpClient functional tests started failing for UWP.